packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/FIATransactionCacheTests.swift

```diff
@@ -0,0 +1,63 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import XCTest
+
+@testable import in_app_purchase_storekit
+
+final class FIATransactionCacheTests: XCTestCase {
+
+  func testAddObjectsForNewKey() {
+    let dummyArray: [Int] = [1, 2, 3]
+    let cache = FIATransactionCache()
+    cache.add(dummyArray, for: TransactionCacheKey.updatedTransactions)
+
+    XCTAssertEqual(
+      dummyArray, cache.getObjectsFor(TransactionCacheKey.updatedTransactions) as! [Int])
```

Comment:
can you use that unwrap function and avoid `!` here? 

```
let updatedTransactions = try XCTUnwrap(cache.getObjectsFor(TransactionCacheKey.updatedTransactions) as? [Int])
```

Reply:
ooh yes, done